packages/components-native/src/StatusIndicator/index.ts

```diff
@@ -0,0 +1,2 @@
+export { StatusIndicator } from "./StatusIndicator";
+export type { StatusType } from "./StatusIndicator";
```

Comment:
I'm not 1000% sure I need this export but I wasn't sure if there would be any knock-in effects of removing it